base/base.go

```diff
@@ -61,7 +61,7 @@ func ParseTime(s string) (time.Time, error) {
 	for _, f := range timeFormats {
 		t, err = time.Parse(f, s)
 		if err == nil {
-			return t, err
+			return t, nil
```

Comment:
Err is nil here. What is the difference between return nil and return err which is nil?